sapi/phpdbg/phpdbg.c

```diff
@@ -911,19 +911,19 @@ void phpdbg_register_file_handles(void) /* {{{ */
 	php_stream_to_zval(s_err, &zerr);
 
 	ic.value = zin;
-	ZEND_CONSTANT_SET_FLAGS(&ic, CONST_CS, 0);
+	ZEND_CONSTANT_SET_FLAGS(&ic, 0, 0);
```

Comment:
not needed

Reply:
I was confused that I don't need to execute this function, but actually it's needed. I changed it `Z_CONSTANT_FLAGS(ic.value) = 0;`. It took me some time to found out how it works, but eventually it works.